### PR TITLE
Revive pass plugin

### PR DESCRIPTION
**Pull request by maxmil**
Take the pass python plugin from the archive and upgrade it to the API 0.5

Bump the plugin version from 0.4.1 to 0.5

### CODE REVIEW

File: pass/__init__.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+
+"""Manage passwords.
+
+This is a 'pass' wrapper extension.
+
+Synopsis:
```
Comment (ManuelSchneid3r) [inline pass/__init__.py:7]: remove the docstring if it does not say more than description() and synopsis()

File: pass/__init__.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    <trigger> generate <location>
+    <trigger> <filter>"""
+
+import fnmatch
+import os
+from albert import *
+
+md_iid = "0.5"
+md_version = "0.5"
+md_name = "Pass"
```
Comment (ManuelSchneid3r) [inline pass/__init__.py:17]: please add a md_license and add yourself to md_maintainers (if you want to be a maintainer)

File: pass/__init__.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+                    completion="pass %s" % password,
+                    actions=[
+                        Action("copy", "Copy", lambda pwd=password: runDetachedProcess(["pass", "--clip", pwd])),
+                        Action("edit", "Edit", lambda pwd=password: runDetachedProcess(["pass", "edit", pwd])),
+                        Action("remove", "Remove", lambda pwd=password: runDetachedProcess(["pass", "rm", "--force", pwd])),
+                    ]
+                ),
+            )
+
+        return query.add(results)
```
Comment (ManuelSchneid3r) [inline pass/__init__.py:87]: is this return intended?

**Review by ManuelSchneid3r — CHANGES_REQUESTED** [review]

File: pass/__init__.py (hunk 1 above)
Comment (maxmil) [reply pass/__init__.py]: Removed

File: pass/__init__.py (hunk 2 above)
Comment (maxmil) [reply pass/__init__.py:9]: Added myself and BSD-3

File: pass/__init__.py (hunk 3 above)
Comment (maxmil) [reply pass/__init__.py]: No. Removed

File: pass/__init__.py [hunk 4]
```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+
+import fnmatch
+import os
+from albert import *
+
+md_iid = "0.5"
+md_version = "0.5"
```
Comment (ManuelSchneid3r) [inline pass/__init__.py:8]: Make the version 1.2 before the interface id and plugin version has been one. 0.4 was the interface '.1' was the plugin version. Now it is iid 0.5 and version (increased) 1.2

**Comment by ManuelSchneid3r** [comment]
Sorry for this iterative review

**Pull request merged**